x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/summary_section.tsx

```diff
@@ -14,51 +14,75 @@ import { CspCounterCard, CspCounterCardProps } from '../../../components/csp_cou
 import { CompactFormattedNumber } from '../../../components/compact_formatted_number';
 import { ChartPanel } from '../../../components/chart_panel';
 import { CloudPostureScoreChart } from '../compliance_charts/cloud_posture_score_chart';
-import type { ComplianceDashboardData, Evaluation } from '../../../../common/types';
+import type {
+  ComplianceDashboardData,
+  Evaluation,
+  PosturePolicyTemplate,
+} from '../../../../common/types';
 import { RisksTable } from '../compliance_charts/risks_table';
 import {
   useNavigateFindings,
   useNavigateFindingsByResource,
 } from '../../../common/hooks/use_navigate_findings';
-import { RULE_FAILED } from '../../../../common/constants';
+import {
+  CSPM_POLICY_TEMPLATE,
+  KSPM_POLICY_TEMPLATE,
+  RULE_FAILED,
+} from '../../../../common/constants';
 
 export const dashboardColumnsGrow: Record<string, EuiFlexItemProps['grow']> = {
   first: 3,
   second: 8,
   third: 8,
 };
 
-export const CloudSummarySection = ({
+// TODO: CIS AWS - replace query to use policy_template field when available
+export const getPolicyTemplateQuery = (policyTemplate: PosturePolicyTemplate) => {
+  if (policyTemplate === CSPM_POLICY_TEMPLATE) return { 'rule.benchmark.id': 'cis_aws' };
+  if (policyTemplate === KSPM_POLICY_TEMPLATE) return { 'rule.benchmark.id': 'cis_k8s' };
+  return {};
+};
```

Comment:
```suggestion
export const getPolicyTemplateQuery = (policyTemplate: PosturePolicyTemplate) => {
  switch (policyTemplate) {
    case CSPM_POLICY_TEMPLATE:
      return { 'rule.benchmark.id': 'cis_aws' };
    case KSPM_POLICY_TEMPLATE:
      return { 'rule.benchmark.id': 'cis_k8s' };
  }

  assertNever(policyTemplate)
};
```

IMO in union types a switch case makes a good fit. the `assertNever` makes the type checking exhaustive, which means if we add another member to `PosturePolicyTemplate` TS will complain here, and in runtime will throw if we get there. if you'd like to read more about it, the TS team lead wrote about it in [stackoveflow](https://stackoverflow.com/questions/39419170/how-do-i-check-that-a-switch-block-is-exhaustive-in-typescript)

no need to return an empty object either, as that code branch is `never`

Reply:
thats a really good text-book example for when to use never, i agree.
but i did some thinking ahead here, the next dashboard is the `overview` dashboard which is some combination of the current ones. suggestively, it should have no filter for the benchmark since it displays all of them, so the empty object made sense. 

there are other options to do that of course, wdyt? good enough for now or should be addressed?